.gitignore

```diff
@@ -1,6 +1,391 @@
-node_modules
-yarn-error.log
+# Created by https://www.toptal.com/developers/gitignore/api/linux,macos,windows,node,yarn,visualstudiocode,jetbrains,vim
```

Comment:
Not all the things in the generated file make sense. E.g. there's no need to target some specific files in `.idea` dir, should be ignored all together. If idea adds smth new, it's gonna make it to the codebase.

Also credit to toptal doesn't seem right here. Please remove it.

Reply:
why does credit to toptal seem incorrect? that is the tool used to quickly generate a comprehensive .gitignore file.

Reply:
I've added some arguments in my 1st comment. 

- I believe there are a few incorrect cases.
- Referencing a 3rd party service in our codebase doesn't feel right to me. It's useful ok, however it's not a node_module and it's actually a live file that is maintained by us